src/ui/components/DropdownMenu/styles.scss

```diff
@@ -6,8 +6,8 @@
   @include text-align-start();
 
   display: inline-block;
-  padding-bottom: 6px;
-  padding-top: 8px;
+  padding-bottom: 2px;
+  padding-top: 4px;
```

Comment:
I should have left a comment obviously. The `padding-bottom` was important to avoid the menu to auto-close itself (because of focus lost) when user moves the cursor from username to the menu itself.

Reply:
I tested locally and it is all good.